### ISSUE TITLE
[Q] Huge .wandb file?

### ISSUE THREAD

**Issue by MauritsBleeker** [wandb/client#2292]
![Schermafbeelding 2021-06-15 om 10 31 11](https://user-images.githubusercontent.com/44800747/122020165-d07bf200-cdc4-11eb-855c-6eb6d2da7586.png)

Hello all,

I have a question about the *.wand file in the local .wandb folder. In my opinion, it is way too big for what it should be? It is ~ 6GB, for only one experiment/run.

I log some statistics every 10th iteration, including the gradients. Are the gradients the reasons that my log file is so extremely big (the models I use are also not that complex)? I know this depends completely on your experiment setting(s) (how long you train and what you log etc.). But what should I do to keep this file of reasonable size?

Thanks!

Maurits

**Comment by MauritsBleeker** [comment]
Also, is there a (within wandb) way to delete the local file when training is done?

**Comment by vanpelt** [comment]
`.wandb` contains a detailed transaction log of all data logged.  It's usually in the 10's to 100's of megabytes.  We generally suggest you log metrics no more frequently than once per second.   Depending on the speed of your network, 10 forward passes is likely too frequent.  Gradients should generally be logged once every 10-60 seconds as it will add overhead to your training script to pass that much information between the CPU and GPU.  You can specify this with the `log_freq` argument of `wandb.watch`.

To clean local directories you can run `wandb sync --clean` which defaults to removing directory older than 24 hours.  You can also specify --clean-old-hours if you want to remove more recent runs.

**Comment by MauritsBleeker** [comment]
Thanks
